object-detector.yaml

```diff
@@ -1,30 +0,0 @@
-apiVersion: v1
-kind: Pod
-metadata:
-  name: object-detector
-  namespace: default
-  labels:
-    app: object-detector
-spec:
-  restartPolicy: Always
-  containers:
-  - env:
-    name: object-detector
-    image: codait/max-object-detector
----
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: object-detector
-    component: object-detector
-  name: object-detector
-spec:
-  ports:
-  - name: http
-    port: 5000
-    targetPort: 5000
-  selector:
-    app: object-detector
-  sessionAffinity: None
-  type: NodePort
```

Comment:
Not being a NodePort, how are you accessing it (e.g. from Curl)? Are you requiring ingress to be configured?

Reply:
A `LoadBalancer` enables and utilizes the `NodePort` while also giving the user access to the API on port `5000` instead of needing to look up the `NodePort` port. From the little I read, ingress seems to be a really powerful implementation of a loadbalence, but I'm just using the normal Kube version. I followed this doc (and the links in it) to make these updates https://kubernetes.io/docs/concepts/configuration/overview/

Reply:
@lresende nix that, after @ptitzler found out that `LoadBalancer`s aren't suported by IBM Cloud's free their we decided at scrum to just use `NodePort`